### DISCUSSION TITLE
how to change the request url

### DISCUSSION THREAD

**Discussion by luqiang2013** [Kong/kong#9759] [Help]
some urls like :
```
http://example.com?actual=http%3A%2F%2Fother.com%2F
```
how can i go to the `http%3A%2F%2Fother.com%2F`, when i  go to `http://example.com`

http 302 is denied...

**Comment by StarlightIbuki** [comment]
Not quite sure what you mean by
> how can i go to the http%3A%2F%2Fother.com%2F, when i go to http://example.com

and why 302 is denied.

Could you explain in detail?
